packages/syntax-core/src/Checkbox/Checkbox.tsx

```diff
@@ -68,6 +70,12 @@ const Checkbox = ({
    * @defaultValue false
    */
   error?: boolean;
+  /**
+   * Whether or not there is an error with the input
```

Comment:
@somethiiing the description doesn't make a lot of sense - can we make it more specific?

Reply:
oops, i had copied and pasted the above error to get the same formatting, but forgot to change the actual error message